web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -12,6 +12,22 @@ function isR2L() {
     return !!$('html[dir=rtl]').length;
 }
 
+function make_multi(id) {
```

Comment:
Why was this factored out of `fixmystreet.set_up`? Was it not in scope when `admin_page_form` was being called?